integration_tests/features/WalletCli.feature

```diff
@@ -0,0 +1,115 @@
+Feature: Wallet CLI
+
+    Scenario: As a user I want to change base node for a wallet via command
+        Given I have a base node NODE1 connected to all seed nodes
+        And I have a base node NODE2 connected to all seed nodes
+        And I have wallet WALLET connected to base node NODE1
+        Then I change base node of WALLET to NODE2
```

Comment:
We need to add a check here to see if the base_node was changed. 
For example:
`Then wallet WALLET is connected tp NODE2`

Reply:
I did had that originally. It would be nice to have but I had remove it this morning, because it somehow caused the test to be flaky (I was not able to reproduce that in my local environment). It failed set base node, and set custom base node. One test never connected and one test did connect but the node reported not connected. I can create a ticket to upgrade these test and a TODO in the feature file. But I don't think it's worth investing effort to fix this issue. Let me know what you think. We can discuss this tomorrow morning.